build_tools/travis/before_install.sh

```diff
@@ -28,6 +28,10 @@ if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
         brew install vtk
     fi
 
+    if [[ "$USE_OMP" == 1 ]]; then
+        brew install llvm
```

Comment:
>CMake Error at /usr/local/Cellar/cmake/3.8.2/share/cmake/Modules/CMakeTestCCompiler.cmake:51 (message):
  The C compiler
  "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang"
  is not able to compile a simple test program.
  It fails with the following output:
   Change Dir: /Users/travis/build/openmeeg/openmeeg/build/OpenMEEG/build/CMakeFiles/CMakeTmp
  /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang
  -fopenmp=libomp -w -o CMakeFiles/cmTC_f4998.dir/testCCompiler.c.o -c
  /Users/travis/build/openmeeg/openmeeg/build/OpenMEEG/build/CMakeFiles/CMakeTmp/testCCompiler.c
  clang: error: unsupported argument 'libomp' to option 'fopenmp='

for now I have that, while I've set:
        export CXX="/usr/local/opt/llvm/bin/clang"
        export CPPFLAGS="$CPPFLAGS -I/usr/local/opt/llvm/include -fopenmp"
        export LDFLAGS="$LDFLAGS -L/usr/local/opt/llvm/lib"

I'm trying on our ci mac machine... but it takes ages to brew llvm